ansible/roles/redpanda/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+storage_path: /mnt/vectorized
```

Comment:
I think it would be better to use actual product name here (redpanda instead of vectorized). Change my mind :P